docs/en/Use-Cases.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: global
+title: Use Cases
+group: Home
+---
+
+* Table of Contents
+{:toc}
+
+
+Here is a description of a few common use cases of Alluxio.
+
+## A Distributed Caching Layer for Data on Object Storage
+
+In many organizations, an increasingly popular architecture is to leverage the
+object storage as the single source of data for bigdata analytics jobs like
+Hadoop, Spark or Presto. Object storage can be public object storage services
+(e.g., AWS S3, Azure blob storage, Google Cloud Storage, Aliyun OSS, Tencent COS
+and etc) or the on-premise object stores (e.g., Ceph, Swift and etc).
+
+Though object stores are often more cost-effective, scalable, available and
+easier to maintain, they are not a "real" filesystem for a few reasons even
+they may provide a filesystem-like interface:
+
+- Lack of filesystem-level data caching so multiple jobs accessing the
+ same data set can benefit from caching the frequently accessed common
+ data.
+- Lack of node-level data locality on computation as data is always read
+ remotely and never node-local to the compute tasks.
+- Operations to list objects can be very slow especially when there is a massive
+ amount of objects in the bucket.
+- Operations to rename objects can be very slow and get weaker
+ consistency guarantees.
```

Comment:
Can add throughput limitations (network bandwidth limit) and also security model is different from file systems.

Reply:
added